chaoscenter/web/src/views/ChaosHub/__tests__/ChaosHub.test.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { render, fireEvent, screen } from '@testing-library/react';
+import { ApolloError } from '@apollo/client';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { TestWrapper } from 'utils/testUtils';
+import ChaosHub from '../ChaosHub';
+
+const queryClient = new QueryClient();
```

Comment:
Please refrain from instantiating new clients in the test itself, that leads to multiple open clients and is a memory leak. Use the <TestWrapper> pattern instead where <ReactQueryClient> is already wrapped.